WordPress/src/main/res/drawable/ic_prepublishing_close_grey_24dp.xml

```diff
@@ -0,0 +1,8 @@
+<!-- drawable/close.xml -->
```

Comment:
Have you considered using `tint` instead of introducing a new drawable with a different color. I believe "all" our single-color-drawables are always white and we use `tint` to set their color. Wdyt?

Reply:
Thanks for this. I never realized that this was the approach being used. Very creative 😄 3cceb1694865fffe77797b394e56795a93cd0883